tools/js-sdk-release-tools/src/common/migration.ts

```diff
@@ -0,0 +1,28 @@
+import { logger } from "../utils/logger";
+import { runCommand, runCommandOptions } from "./utils";
+import { getNpmPackageInfo } from "./npmUtils";
+import { ensureDir } from "fs-extra";
+import { posix } from "path";
+
+// TODO: remove when emitter is ready
+export async function migratePackage(packageDirectory: string): Promise<void> {
```

Comment:
should we detect if the module is already esm?

Reply:
e.g. https://github.com/Azure/azure-sdk-for-js/pull/32201/commits/33f43d4e0592e41c3ba1667bcd2210a7ce224f03

Reply:
and they are idemptent

Reply:
Got it, thanks for the info and pointers.